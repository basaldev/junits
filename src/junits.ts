import Hizuke from './hizuke';

export const hizuke = Hizuke;
