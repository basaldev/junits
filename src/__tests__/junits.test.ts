import Hizuke from '../hizuke';
import * as jUnits from '../junits';

describe('jUnits', () => {
  let expectedReturn;

  beforeEach(() => {
    expectedReturn = {
      hizuke: Hizuke
    };
  });

  it('should be defined', () => {
    expect(jUnits).toMatchSnapshot(expectedReturn);
  });
});
