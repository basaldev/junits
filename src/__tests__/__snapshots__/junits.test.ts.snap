// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`jUnits should be defined 1`] = `
Object {
  "hizuke": [Function],
}
`;
